src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -85,6 +85,16 @@ public Path getAddressBookFilePath() {
         return model.getAddressBookFilePath();
     }
 
+    @Override
+    public Path getTaskBookFilePath() {
+        return model.getTaskBookFilePath();
+    }
+
+    @Override
+    public Path getOrderBookFilePath() {
+        return model.getOrderBookPath();
```

Comment:
Would it be better to rename the method for consistency with `AddressBook` and `TaskBook`?
```suggestion
        return model.getOrderBookFilePath();
```